src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Consent/HealthCertificateExportCertificatesInfoViewController.swift

```diff
@@ -3,15 +3,23 @@
 //
 
 import UIKit
+import PDFKit
 
 class HealthCertificateExportCertificatesInfoViewController: DynamicTableViewController, FooterViewHandling, DismissHandling {
 	
 	// MARK: - Init
 	
 	init(
-		viewModel: HealthCertificateExportCertificatesInfoViewModel
+		viewModel: HealthCertificateExportCertificatesInfoViewModel,
+		onDismiss: @escaping CompletionBool,
+		onTapContinue: @escaping (PDFDocument) -> Void,
```

Comment:
Optional:
For closure types, that comes very often, it could be useful to define a new type alias like `CompletionBool`.
Do you thing `CompletionPDFDocument` could be a candidate?

Reply:
Nice idea. However, it's only used once, I would prefer to stay with it.